locale/en/manageiq.po

```diff
@@ -72385,12 +72598,30 @@ msgid ""
 msgstr ""
 
 #:
-#: ../app/models/manageiq/providers/awx/automation_manager/azure_credential.rb
+#: ../app/models/manageiq/providers/embedded_terraform/automation_manager/ibm_cloud_credential.rb
+msgid ""
+"The API key for IBM Cloud Classic Infrastructure A value for this field is req"
+"uired when using classic IBM Cloud resources. A valid connection must have val"
+"ue for IBM Cloud API Key or, IBM Cloud Classic Infrastructure User Name and IB"
+"M Cloud Classic Infrastructure API Key."
+msgstr ""
+
+#:
+#: ../app/models/manageiq/providers/embedded_terraform/automation_manager/ibm_cloud_credential.rb
+msgid ""
+"The API key for IBM Cloud. A value for this field is required if classic user "
+"name and classic API key are not provided. A valid connection must have value "
+"for IBM Cloud API Key or, IBM Cloud Classic Infrastructure User Name and IBM C"
+"loud Classic Infrastructure API Key."
+msgstr ""
+
+#:
```

Comment:
simplified in https://github.com/ManageIQ/manageiq-providers-embedded_terraform/pull/21